packages/expo-image/src/Image.tsx

```diff
@@ -18,10 +18,18 @@ export class Image extends React.PureComponent<ImageProps> {
 
   /**
    * Preloads images at the given urls that can be later used in the image view.
-   * Preloaded images are always cached on the disk, so make sure to use
-   * `disk` (default) or `memory-disk` cache policy.
+   * Preloaded images are always cached on the disk, so make sure to use disk
+   * (default) or memory-disk cache policy.
+   *
+   * @param urls - A url string or an array of urls of images to prefetch.
+   * @param cachePolicy - The cache policy for pre-fetched images. Defaults to
+   * disk
+   * @return A promise resolving to `true` as soon as all images have been
+   * successfully prefetched. If an image fails to be prefetched, the promise
+   * will immediately resolve to `false` regardless of whether other images have
+   * finished prefetching.
```

Comment:
```suggestion
   * Preloaded images are always cached on the disk, so make sure to use `disk`
   * (default) or `memory-disk` [cache policy](#cachepolicy).
   *
   * @param urls - A url string or an array of URLs of images to prefetch.
   * @param cachePolicy - The cache policy for pre-fetched images. Defaults to
   * `disk`.
   * @return A promise resolving to `true` as soon as all images have been
   * successfully prefetched. If an image fails to be prefetched, the promise
   * will immediately resolve to `false` regardless of whether other images have
   * finished prefetching.
```

Let's use inline syntax highlights for type names and urls to URLs for correct usage.